app/lib/LevelBus.coffee

```diff
@@ -198,8 +198,9 @@ module.exports = class LevelBus extends Bus
     @changedSessionProperties.state = true
     @saveSession()
 
-  onVictory: ->
+  onVictory: (e) ->
     return unless @onPoint()
+    return if e and e.capstoneVictory
```

Comment:
Is there a better way to handle this "done but not really done" concept? 

When capstone level goals are being completed, it should behave like the level is being completed in _most ways_. However, the level is not truly complete until all the additional goals have been completed.

Reply:
Do we not want to set complete = true on full completion of the capstone level?

Reply:
This will be set to `complete = true` when we don't pass in this option. That is only meant to happen when all the goals are complete (was missing, is in now).